config/kibana.yml

```diff
@@ -102,3 +102,10 @@
 # The default locale. This locale can be used in certain circumstances to substitute any missing
 # translations.
 #i18n.defaultLocale: "en"
+
+
+optimize:
```

Comment:
If you put this into `kibana.dev.yml`, it will load for your but not show up here, which is what we want. start kibana with `node scripts/kibana --dev`